src/Microsoft.Kusto.ServiceLayer/Localization/sr.cs

```diff
@@ -1,5 +1,5 @@
 // WARNING:
-// This file was generated by the Microsoft DataWarehouse String Resource Tool 5.0.0.0
+// This file was generated by the Microsoft DataWarehouse String Resource Tool 4.0.0.0
```

Comment:
I tried to build with latest `main` and still got this change.

Reply:
manually revert it for now.